python/pyspark/ml/regression.py

```diff
@@ -447,7 +447,7 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeModel(JavaModel):
+class DecisionTreeModel(JavaModel, MLWritable, MLReadable):
     """Abstraction for Decision Tree models.
```

Comment:
@Wenpei Please check wether it supports ```save/load``` for the peer Scala implementation. Some algorithms such as ```DecisionTree``` did not support it currently. And you should add doc test that will test the correctness of your modification.